package/src/bin/deployment-example.toml

```diff
@@ -0,0 +1,42 @@
+# This manifest describes the servers that omicron will be installed to, along
+# with any ancillary information specific to a given server.
+#
+# It is ingested by the `thing-flinger` tool.
+
+# This must be an absolute path
+local_source = "/Users/ajs/oxide/omicron"
+
+[builder]
+# `server` must refer to one of the `servers` in the servers table
+server = "atrium"
+omicron_path = "/home/andrew/oxide/omicron"
+git_treeish = "thing-flinger"
```

Comment:
This is definitely Git jargon, I might suggest calling it subdirectory or something more obvious.

Reply:
I'll add a comment.